util/middleware.js

```diff
@@ -19,10 +19,8 @@ export async function secureFetch (url, accessToken, method = 'get', options = {
     headers,
     ...options
   })
-  return {
-    data: await res.json(),
-    timestamp: new Date()
-  }
+  const json = await res.json()
```

Comment:
When merging with `dev`, please update the calls to `secureFetch` in `LayoutWithAuth0` to reflect this change. See https://github.com/ibi-group/otp-admin-ui/blob/dev/components/LayoutWithAuth0.js#L74